### PR TITLE
Add circleci

### PR DESCRIPTION
**Pull request by xorima**
Adds circleci for testing 

### Description

[Describe what this change achieves]

### Issues Resolved

[List any existing issues this PR resolves]

### Contribution Check List

- [ ] All tests pass.
- [x] New functionality includes testing.
- [x] New functionality has been documented in the README if applicable

### CODE REVIEW

**Comment by kitchen-porter** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Error">
          1 Error
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">Please add labels to this Pull Request</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

**Review by damacus — COMMENTED** [review]
Merge conflict?

**Comment by Xorima** [comment]
not a clue how, my laptop has a merge conflict with itself ... now just waiting for lint to run, seems to be stuck for some reason

File: Dangerfile [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+fail 'Please provide a summary of your Pull Request.' if github.pr_body.length < 10
+
+fail 'Please add labels to this Pull Request' if github.pr_labels.empty?
+
+warn 'This is a big Pull Request.' if git.lines_of_code > 400
+
+# Require a CHANGELOG entry for non-test changes.
+if !git.modified_files.include?('CHANGELOG.md') && code_changes?
+  fail 'Please include a [CHANGELOG](https://github.com/sous-chefs/line-cookbook/blob/master/CHANGELOG.md) entry.'
```
Comment (damacus) [inline Dangerfile]:
```suggestion
  fail 'Please include a [CHANGELOG](https://github.com/sous-chefs/swap/blob/master/CHANGELOG.md) entry.'
```

**Review by gsreynolds — APPROVED** [review]

**Pull request merged**